rosetta/translate_utils.py

```diff
@@ -167,3 +170,31 @@ def translate_by_google(
         raise TranslationException("Google API error: {}".format(e))
     else:
         return str(api_response.translations[0].translated_text)
+
+
+def translate_by_openai(text: str, from_language: str, to_language: str, api_key: str) -> str:
+    """
+    Translate text using OpenAI's GPT-3.5-turbo-instruct engine.
+    param text: The text to translate.
+    param from_language: The language of the text.
+    param to_language: The language to translate the text into.
+    param api_key: The OpenAI API key.
+    return: The translated text.
+    """
+
+    import openai
```

Comment:
What library is this? :pray:

Reply:
https://pypi.org/project/openai/0.26.5/

This library provides convenient access to the OpenAI API

Reply:
Any good reason to mention 0.28.0, when the latest version is 1.35.3?

Reply:
Upgraded 🙏

Reply:
Awesome, thanks.